src/app.ts

```diff
@@ -17,12 +19,8 @@ export class App {
             .map(ReviewConverter.convertToReview)
             .sort((a, b) => (a.date < b.date) ? 1 : -1);
 
-        fs.writeFile('reviews.json', JSON.stringify(snippets), (err) => {
-            if (err) {
-                // tslint:disable-next-line
-                console.log(err)
-            }
-        });
+        this.mongoDBService.connectDB();
+        this.mongoDBService.populateDB(snippets);
```

Comment:
Might be easier to use if populateDB connected for the user, if it needed to, rather than the consumer having to connect before issuing commands.